src/_includes/macros/form.html

```diff
@@ -0,0 +1,244 @@
+{# ==========================================================================
+
+   form.input()
+
+   ==========================================================================
+
+   Description:
+
+   Builds form text field.
+
+   label: Name of the field.
+
+   type: Type of field. Will default to text, but can change to other
+         types to take advantage of HTML5 validation
+          Possible fields: text, tel, email, url, number, date.
```

Comment:
Trivial: extra indent space here and throughout. Could possibly left-align the values too, like https://github.com/cfpb/cfgov-refresh/blob/flapjack/src/_includes/macros/active-filters.html#L21-L28